packages/filezilla.rb

```diff
@@ -31,6 +29,7 @@ class Filezilla < Package
   depends_on 'sqlite'
   depends_on 'wxwidgets'
   depends_on 'xdg_utils'
+  depends_on 'libjpeg_turbo'
```

Comment:
The problem with libjpeg (no _turbo) is it doesn't provide libjpeg.so.8 which filezilla requires.  The current version is libjpeg.so.9.  I guess we can create a symlink from libjpeg.so.9 to libjpeg.so.8 to see if that works and then remove the libjpeg_turbo package to avoid any future conflicts.